### ISSUE TITLE
Help in webapp setup

### ISSUE THREAD

**Issue by dasarisrinivas** [Intelehealth/intelehealth-doctor-webapp#87]
Please let us know the id and password for default or please let us know the flow diagram

**Comment by AmirSmarty** [comment]
Username and Password please?

**Comment by dasarisrinivas** [comment]
Yes able to launch login page and it’s calling autenticate service demo api .. does it has any default username and password to login

**Comment by rishavk99** [comment]
??

**Comment by danmaby** [comment]
@kundanmail55 what is the default Username/Password for a fresh install?  

![Screenshot 2020-07-26 at 15 36 00](https://user-images.githubusercontent.com/13104185/88482035-a1fbeb00-cf56-11ea-8ab8-1736a6f24352.png)

**Comment by heyitskundan** [comment]
@danmaby admin/Admin123

**Comment by kaiz3n77** [comment]
The admin/Admin123 is only working with the demo.intelehealth.io but when I ran the web app on the local machine, I could only get up to the login screen (from localhost:4200), admin/Admin123 is not working, click on Login to Dashboard does nothing(It does not show wrong user/password or anything)

**Comment by harshit211997** [comment]
@kaiz3n77 Can you please see if #101 helps?
